dm/master/server.go

```diff
@@ -1332,6 +1346,62 @@ func (s *Server) generateSubTask(ctx context.Context, task string) (*config.Task
 	return cfg, stCfgs, nil
 }
 
+func (s *Server) removeMetaData(ctx context.Context, cfg *config.TaskConfig) error {
```

Comment:
Is there any valid unit/integration test case?

Reply:
Added a UT in https://github.com/pingcap/dm/pull/651/commits/0713a35d30b9c9a40fc6a559868689f52a7680e2.